hs_access_control/tests/test_group_membership_request.py

```diff
@@ -446,6 +446,42 @@ def test_user_sending_request_with_explanation(self):
         self.assertEqual(
             self.modeling_group.gaccess.group_membership_requests.count(), 0)
 
+    def test_user_sending_invitation_without_explanation(self):
+        # require explanation on the group
+        self.modeling_group.gaccess.requires_explanation = True
+
```

Comment:
Though it works based on how the function `create_group_membership_request()`  uses the parameters, I would prefer to save the changes to `gaccess` to db.